src/__tests__/core/signTransactionCapricoin.spec.js

```diff
@@ -84,7 +84,8 @@ const oneTwoFee = (): SubtestSignTransaction => {
     const expectedResponses: Array<ExpectedSignTransactionResponse> = [
         {
             payload: {
-                serializedTx: '0100000034b1cd5b010fbf9f7826f5cac19d58aef797767255433b767ab5e00d122c916229a54c822e000000006b483045022100ccc4e840995a5af9866897f02f957c0f9545fce07681210cc51853f7a4ce96c30220483afbcbe24d63d98fd81752f6e2b56de1e224eca11815a60c466cd2347d61670121021fcf98aee04939ec7df5762f426dc2d1db8026e3a73c3bbe44749dacfbb61230ffffffff02184f0700000000001976a914b5fcce71b52fe2a05479610906e6aa81f4a6e76488ac40420f00000000001976a914544e87b53141b883b392d114168c5403899756c488ac00000000',
+                // serializedTx: '0100000034b1cd5b010fbf9f7826f5cac19d58aef797767255433b767ab5e00d122c916229a54c822e000000006b483045022100ccc4e840995a5af9866897f02f957c0f9545fce07681210cc51853f7a4ce96c30220483afbcbe24d63d98fd81752f6e2b56de1e224eca11815a60c466cd2347d61670121021fcf98aee04939ec7df5762f426dc2d1db8026e3a73c3bbe44749dacfbb61230ffffffff02184f0700000000001976a914b5fcce71b52fe2a05479610906e6aa81f4a6e76488ac40420f00000000001976a914544e87b53141b883b392d114168c5403899756c488ac00000000',
```

Comment:
Is there a reason why we would like to keep the old value in the comment here? If not, let's remove this.